src/js/utils/DOM.js

```diff
@@ -31,6 +36,11 @@ export default {
     if (result.length === 0) {
       result.push(document);
     }
+    if (window.innerWidth <= Responsive.smallSize()
+      && result[0].classList
+      && result[0].classList.contains(LAYER_CLASS)) {
+      result = [document];
```

Comment:
Found a cleaner & simpler fix... Setting `overflow: auto` to `.grommetux-layer` allows it to be scrollable.